extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/PeertubeService.java

```diff
@@ -59,7 +61,12 @@ public ListLinkHandlerFactory getCommentsLHFactory() {
 
     @Override
     public SearchExtractor getSearchExtractor(SearchQueryHandler queryHandler) {
-        return new PeertubeSearchExtractor(this, queryHandler);
+        final List<String> contentFilters = queryHandler.getContentFilters();
+        boolean external = false;
+        if (contentFilters.size() > 0 && contentFilters.get(0).startsWith("sepia_")) {
```

Comment:
Why only `startsWith("sepia_")` and not `equals(SEPIA_VIDEOS)`?

Reply:
because if we implement other search filters (e.g. channels), it will not change, the baseUrl with sepia filters will still be sepiasearch.org (by default, unless another baseUrl for another index is provided)

Reply:
as I used the same convention as YoutubeMusicSearchExtractor, i.e. filters start with music_ when using MusicSearchExtractor